File: HELLO_ANDROID01/app/src/main/java/kr/co/aiai/app/MainActivity06.java
package kr.co.aiai.app;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity06 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main06);

        Button btn=findViewById(R.id.btn);
        btn.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view) {
                myclick();
            }
        });
    }
    void myclick(){
        EditText first=findViewById(R.id.et_first);
        EditText last=findViewById(R.id.et_last);

        String firstt= String.valueOf(first.getText());
        String lastt= String.valueOf(last.getText());

        int ifirst=Integer.parseInt(firstt);
        int ilast=Integer.parseInt(lastt);
        String txt="";
        for(int i=ifirst;i<=ilast;i++){
            for(int j=ifirst;j<ifirst+i;j++){
                txt+="*";
            }
            txt+="\n";
        }
        EditText et_disp=findViewById(R.id.et_disp);

        et_disp.setText(txt);

    }
}

File: HELLO_AI4/ANDROID/app/src/main/java/kr/co/aiai/myapp/MainActivity06.kt
package kr.co.aiai.myapp

import android.app.AlertDialog
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity06 : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main06)
        var btn = findViewById<Button>(R.id.btn)
        btn.setOnClickListener() {
            myclick()
        }

    }

    fun myclick() {
        var first = findViewById<EditText>(R.id.et_first)
        var last = findViewById<EditText>(R.id.et_last)
        var firstt=first.getText().toString()
        var lastt=last.getText().toString()

        var ifirst=Integer.parseInt(firstt)
        var ilast=Integer.parseInt(lastt)
        var txt=""
        for(i in ifirst until ilast){
            for(j in ifirst until ifirst+i){
                txt+="*"
            }
            txt+="\n"
        }

        var et_disp=findViewById<EditText>(R.id.et_disp)

        et_disp.setText(txt)
    }



}